pkg/cli/store.go

```diff
@@ -92,6 +92,8 @@ func init() {
 	set.Bool("service-poll", true, "sets the collector or certifier to polling mode")
 	set.BoolP("poll", "p", false, "sets the collector or certifier to polling mode")
 
+	set.Bool("query-vuln", false, "if enabled, the ingestor will query OSV for vulnerabilities during ingestion")
```

Comment:
can we add a warning here that this will increase ingestion times..

nit: if enabled, the ingestor will query and ingest OSV for vulnerabilities